client/common/directives/ModelFieldFile/ModelFieldFile.js

```diff
@@ -21,7 +21,7 @@ angular.module('dashboard.directives.ModelFieldFile', [
     restrict: 'E',
     template: '<button class="btn btn-default select-file" ng-hide="disabled">Select File</button> \
       <input type="file" ng-file-select="onFileSelect($files)" ng-hide="disabled"> \
-      <span class="file-upload-info" ng-if="filename"><i class="fa {{getFileIcon(filename)}}"></i>&nbsp;&nbsp;{{ filename }}&nbsp;&nbsp;<span ng-if="fileUrl">(<a href="{{fileUrl}}">download</a>)</span></span> \
+      <span class="file-upload-info" ng-if="filename"><i class="fa {{getFileIcon(filename)}}"></i>&nbsp;&nbsp;{{ filename }}&nbsp;&nbsp;<span ng-if="fileUrl">(<a href="{{fileUrl}}">download</a> | <a target="_blank" href="{{previewUrl}}">preview</a>)</span></span> \
```

Comment:
May want to add a check for previewUrl and only display it if there is a previewUrl. This would be desirable for backwards compatibility (not sure if that is a large requirement).